### DISCUSSION TITLE
Omit request headers (but not response headers)

### DISCUSSION THREAD

**Discussion by DavidAccola** [DannyDainton/newman-reporter-htmlextra#404] [Q&A] [answered]
I have a need to hide Request Headers, but still show (some) Response Headers.

I've got a bit of a Goldilocks situation. --reporter-htmlextra-omitHeaders does too much (also hides the Response Headers, which I don't want to do). --reporter-htmlextra-skipHeaders does too little; I can exclude all Request Headers, but an empty Request Headers area still displays and doesn't look so pretty:

![image](https://user-images.githubusercontent.com/62493194/217662254-830bfea8-7219-498a-a4c6-cf2184665159.png)

Is anyone aware of a way to exclude the Request Headers portion of the report (without excluding Response Headers) that I'm not aware of?

If not, could more specific CLI options like "--reporter-htmlextra-omitRequestHeaders" and ""--reporter-htmlextra-omitResponseHeaders" be considered as an enhancement request?

Thanks!

**Comment by w4dd325** [comment, marked as answer]
Hi @DavidAccola

You can edit the template files and then tell htmlextra to use the alternative template.

So if you duplicated the existing template and then comment out the parts than display the request headers.

I had a quick look (its late and I'm tired but...) and I think this is the bit you would need to alter / remove...
![image](https://user-images.githubusercontent.com/86320001/217669054-88bb371f-50e5-4143-ba17-41e07bdf8ed3.png)

Then use the
--reporter-htmlextra-template <path>
Option to output using the new template.

I don't have enough time to play with this now, but hopefully this gets you on the right track... 
Hope this helps.

**Reply by DavidAccola** [reply]
That's what I needed to know! Thank you!
